driver-core/src/main/java/com/datastax/driver/core/querybuilder/BuiltStatement.java

```diff
@@ -293,25 +311,6 @@ public Boolean isIdempotent() {
         return !hasNonIdempotentOps();
     }
 
-    @Override
-    public String toString() {
-        try {
-            if (forceNoValues)
-                return getQueryString();
-            // 1) try first with all values inlined (will not work if some values require custom codecs,
-            // or if the required codecs are registered in a different CodecRegistry instance than the default one)
-            return maybeAddSemicolon(buildQueryString(null, CodecRegistry.DEFAULT_INSTANCE)).toString();
```

Comment:
I know that this behavior already previously existed (for the statement that was actually executed).  The other consideration is that parameters in simple statements are only supported in protocol V2+, although at this point I don't think we need to optimize for protocol V1.  I wonder why we don't just parameterize everything?